recipes/qt/5.x.x/patches/chromium-skia-missing-iterator-include.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/chromium/third_party/skia/src/utils/SkParseColor.cpp b/chromium/third_party/skia/src/utils/SkParseColor.cpp
+index 7260365b2c6..96f46dd67f9 100644
+--- a/chromium/third_party/skia/src/utils/SkParseColor.cpp
++++ b/chromium/third_party/skia/src/utils/SkParseColor.cpp
+@@ -9,3 +0,5 @@
+ #include "include/utils/SkParse.h"
+ 
++#include <algorithm> // std::lower_bound
```

Comment:
You also need an include here for `<iterator>` in order to include `std::begin` and `std::end`. Without that, I hit a compilation error with GCC 12.

Reply:
I'm sorry, but I find this too bleeding edge. gcc 12 is not even released. AFAIK, this fix is not submitted to skia.
Each patch we add to a recipe is a burden we have to carry during the next releases, with the risk that it conflicts with other changes upstream. This is why using patches of commits already merged upstream is the least bad solution: there is a minimal chance of conflict, and we can very easily check in which version the patch is integrated, to drop it as soon as necessary.
also, if you absolutely need to be bleeding edge, then don't use a version of qt which is more than 1 year old ([5.15.3](https://github.com/qt/qtbase/commits/v5.15.3-lts-lgpl)). you should instead use 6.3.0.

Reply:
I submitted a patch upstream which has now been merged: https://skia-review.googlesource.com/c/skia/+/533981

Reply:
thanks, this will ease maintenance !